study-builder/studies/osteo/subs.conf

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
@charlottedye @MocanaAtBroad this is another way we can use `--substitutions` besides centralizing i18n strings, i.e. centralizing some commonly repeated values in the study config/emails.